altair/vegalite/v5/api.py

```diff
@@ -1551,7 +1552,9 @@ def selection_interval(
 
 def selection_point(
     name: str | None = None,
-    value: Optional[Any] = Undefined,
+    value: Optional[
+        PrimitiveValue_T | Temporal | SchemaBase | Sequence[SchemaBase | Map] | Map
+    ] = Undefined,
```

Comment:
### TODO
- [ ] Implement the plan from #3656

Reply:
Resolved in [`30aa4ba` (#3653)](https://github.com/vega/altair/pull/3653/commits/30aa4baa537308ffa7e27effb2bcd386ee9a701e)